corehq/apps/hqcase/_design/views/by_domain_hq_user_id/map.js

```diff
@@ -1,5 +1,5 @@
 function(doc){
     if(doc.doc_type == "CommCareCase" && doc.hq_user_id) {
-        emit([doc.domain, doc.hq_user_id], null);
+        emit([doc.domain, doc.hq_user_id, doc.type], null);
```

Comment:
That's good to know! I've created a new "by_domain_hq_user_id_type". And I've just realised that `get_callcenter_case_mapping()` uses `by_domain_hq_user_id` with just `domain` and `hq_user_id`, so this seems like the way to go.